pkg/diagnostics/diagnostics.go

```diff
@@ -0,0 +1,96 @@
+package diagnostics
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/spf13/cobra"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"strmprivacy/strm/pkg/auth"
+	"strmprivacy/strm/pkg/common"
+	"strmprivacy/strm/pkg/util"
+)
+
+type Metrics struct {
+	K int            `json:"k"`
+	L map[string]int `json:"L"`
+	T float64        `json:"T"`
+	E string         `json:"error"`
+}
```

Comment:
The names are really short, with the explicit json mapper they could be more descriptive. Does the type need to be public?

Reply:
type needs to be public. Can change the names